extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -254,19 +282,28 @@ class AmpAccordion extends AMP.BaseElement {
     } else { // Toggle without animation
       this.mutateElement(() => {
         if (toExpand) {
+          if (!section.hasAttribute('expanded')) {
+            this.triggerExpandEvent_(section);
+          }
           section.setAttribute('expanded', '');
           header.setAttribute('aria-expanded', 'true');
           // if expand-single-section is set, only allow one <section> to be
           // expanded at a time
           if (this.element.hasAttribute('expand-single-section')) {
             this.sections_.forEach(sectionIter => {
               if (sectionIter != section) {
+                if (section.hasAttribute('expanded')) {
```

Comment:
You can also put the next two lines `removeAttribute` and `setAttribute` inside this if check, since you're doing an if check now anyway.